src/main/java/com/yelp/nrtsearch/server/luceneserver/highlights/NRTFastVectorHighlighter.java

```diff
@@ -142,7 +142,11 @@ public String[] getHighlights(
           DEFAULT_ENCODER);
     } catch (RuntimeException runtimeException) {
       logger.warn(
-          "FVH failed creating fragments, and the exception is: ", runtimeException.getMessage());
+          String.format(
```

Comment:
No need for String.format since you're only using logger, and you can also log the complete exception. So something like:
```
logger.warn(
              "FVH failed creating fragments for the luceneDocId: {}, highlight query: {}, exception: ",
              leafDocId + hitLeaf.docBase,
              settings.getHighlightQuery(),
              runtimeException.getMessage(),
              runtimeException);
```